src/main.go

```diff
@@ -1,17 +1,23 @@
 package main
 
 import (
-	"github.com/samber/lo"
 	"log"
 	"onepixel_backend/src/db"
 	"onepixel_backend/src/server"
 )
 
 func main() {
-	app := server.CreateApp()
-
 	// Initialize the database
-	lo.Must(db.InitDB())
+	dbConnection, err := db.InitDB()
+	if err != nil {
+		log.Fatalf("Failed to initialize database: %v", err)
```

Comment:
even without this, the internal error from InitDB is self-descriptive. why just add extra lines of code when not needed

Reply:
Works.